<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iconExport</class>
 <widget class="QDialog" name="iconExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>303</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>303</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>303</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Save TSR Icon</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Icon Preview</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGraphicsView" name="iconPreview">
          <property name="minimumSize">
           <size>
            <width>148</width>
            <height>125</height>
           </size>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="dragMode">
           <enum>QGraphicsView::NoDrag</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="fitXButton">
         <property name="toolTip">
          <string>Fit the animation frame to the icon's width, centering vertically</string>
         </property>
         <property name="text">
          <string>Autofit X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fitYButton">
         <property name="toolTip">
          <string>Fit the animation frame to the icon's height, centering horizontally</string>
         </property>
         <property name="text">
          <string>Autofit Y</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="resetButton">
         <property name="toolTip">
          <string>Clear set values and place animation frame at 0,0 at original size</string>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="toolTip">
          <string>Zoom multiplier for animation frame</string>
         </property>
         <property name="text">
          <string>Scale:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="toolTip">
          <string>Zoom multiplier for animation frame</string>
         </property>
         <property name="minimum">
          <number>-30</number>
         </property>
         <property name="maximum">
          <number>2</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="scaleFacBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Zoom multiplier for animation frame</string>
         </property>
         <property name="text">
          <string>1.00x</string>
         </property>
         <property name="maxLength">
          <number>10</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="toolTip">
          <string>Offset, in pixels, from upper left of image</string>
         </property>
         <property name="text">
          <string>Offset X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="offsetXBox">
         <property name="toolTip">
          <string>Offset, in pixels, from upper left of image</string>
         </property>
         <property name="minimum">
          <number>-999999999</number>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="toolTip">
          <string>Offset, in pixels, from upper left of image</string>
         </property>
         <property name="text">
          <string>Offset Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="offsetYBox">
         <property name="toolTip">
          <string>Offset, in pixels, from upper left of image</string>
         </property>
         <property name="minimum">
          <number>-999999999</number>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="centerButton">
         <property name="text">
          <string>Center</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveIconBtn">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>Save Icon</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelBtn">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
